_data/simple-icons.json

```
@@ -3707,6 +3707,11 @@
             "hex": "EC1C24",
             "source": "https://facepunch.com/img/brand/default-light.svg"
         },
+        {
+            "title": "Fairphone",
+            "hex": "2FA6DF",
```

```suggestion
            "hex": "4495D1",
```
This hex is currently used in their logo.